sonus.js

```diff
@@ -1,29 +1,46 @@
 'use strict'
-
 // Load in smart mirror config
-const config = require(__dirname + "/config.js")
-if(!config || !config.speech || !config.speech.keyFilename || !config.speech.model || !config.language){
-  throw "Configuration Error! See: https://docs.smart-mirror.io/docs/configure_the_mirror.html#speech";
+const fs = require('fs')
+var config = require("./config.json")
+
+
+if (!config || !config.speech || !config.speech.keyFilename || !config.speech.hotwords || !config.general.language) {
+  throw "Configuration Error! See: https://docs.smart-mirror.io/docs/configure_the_mirror.html#speech"
 }
 
+var keyFile = JSON.parse(fs.readFileSync(config.speech.keyFilename, "utf8"))
+var umdl = __dirname + '/node_modules/sonus/resources/snowboy.umdl'
+
 // Configure Sonus
 const Sonus = require('sonus')
 const speech = require('@google-cloud/speech')({
-  projectId: config.speech.projectId,
+  projectId: keyFile.project_id,
```

Comment:
I think this is stale and needs to be reverted (since we are now requiring the json file)